core/common/src/main/java/alluxio/security/authorization/AclEntry.java

```diff
@@ -396,6 +411,9 @@ public Builder setActions(AclActions actions) {
      * @return the builder
      */
     public Builder addAction(AclAction action) {
+      if (mActions == null) {
```

Comment:
Replace 
```
private AclActions mActions;
```
with 
```
private AclActions mActions = new AclActions();
```
instead? Maybe also give default values to all the optional builder parameters (e.g. `private boolean mIsDefault = false;`)?